google-cloud-bigquery/src/main/java/com/google/cloud/bigquery/Table.java

```diff
@@ -139,7 +139,7 @@ public Builder setDefinition(TableDefinition definition) {
     }
 
     @Override
-    public TableInfo.Builder setEncryptionConfiguration(EncryptionConfiguration configuration) {
+    public Builder setEncryptionConfiguration(EncryptionConfiguration configuration) {
```

Comment:
did the return type change here or did you just remove the outer class?

Reply:
@pmakani hi Praful - could you explain why TableInfo need to be removed here please?

Reply:
caught up w Praful -- this is to make the method consistent with the other existing methods which are not using TableInfo as an outer class.

Reply:
Looking at this again. This is not OK absent a major version bump, and it probably doesn't justify that. I would mark this method deprecated and introduce a new method that returns a Table.Builder, but you don't need to delete this one now.